.github/workflows/build.yml

```
@@ -91,21 +105,72 @@ jobs:
           cd artifacts && zip -r "$ARTIFACT_BASENAME-universal.zip" "$APP_NAME.app"
       - name: Verify bundled helper resources
         run: |
+          normalize_identifier_set_json() {
+            python3 - "$1" "$2" <<'PY'
+import json
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)